admin_pages/events/templates/event_tickets_metabox_main.template.php

```diff
@@ -116,7 +119,7 @@
                 </tr>
             </thead>
             <tbody id="all-ticket-rows">
-                <?php echo $ticket_rows; // already escaped ?>
+                <?php echo wp_kses($ticket_rows, AllowedTags::getWithFormTags()); ?>
```

Comment:
this looks great, but because there is the _potential_ to unintentionally strip out content, i would like you to do the following for at least the first few PRs like this:

- switch to master branch

- find exactly where a variable like `$ticket_rows` is being echoed out in the DOM by doing something like:
    ```
    echo '<h1>START</h1>' . $ticket_rows . '<h1>END</h1>';
    ```

- using your browser's dev tools, copy all of the HTML for that part of the DOM into a file named `master.html`

- discard your changes

- switch to this branch

- find that same code in DOM again (add your START & END markers if need be)

- copy all of the HTML for that part of the DOM into a file named `kses.html`

- use your IDE or Git Client to perform a file compare or diff on those two files to ensure that nothing is being unintentionally stripped out 

i know that's a huge PITA (pain in the 4$$) but it's the only way for us to know for sure if something is being messed up.

An alternate approach is to backtrace all of the variables and find every bit of code that is feeding content to the template and check each and every bit of HTML to see what attributes are being used, but that seems like way more work to me